pkg/api/customization/feature/feature.go

```diff
@@ -40,7 +40,13 @@ func Formatter(request *types.APIContext, resource *types.RawResource) {
 	if request.Method == http.MethodGet {
 		if resource.Values["value"] == nil {
 			// if value is nil, then this ensure default value will be used
-			resource.Values["value"] = resource.Values["default"]
+			val := false
+			status, ok := resource.Values["status"].(map[string]interface{})
+			if ok {
+				val, _ = status["default"].(bool)
```

Comment:
bool type is guaranteed here ?

Reply:
as long as _ is there, it will use the zero value (false) if it is not a bool.